site/src/Librecores/ProjectRepoBundle/Entity/Project.php

```diff
@@ -323,6 +325,47 @@ public function __construct()
         $this->classifications = new ArrayCollection();
     }
 
+    /**
+     * Normalizes the object into an array of scalars|arrays.
+     *
+     * @param NormalizerInterface $serializer The serializer is given so that you
+     *                                        can use it to serialize objects contained within this object
+     * @param string|null         $format     The format is optionally given to be able to serialize differently
+     *                                        based on different output formats
+     * @param array               $context    Options for serializing this object
+     *
+     * @return array|string|int|float|bool
+     */
+    public function normalize(NormalizerInterface $serializer, $format = null, array $context = array())
+    {
+        return [
+            'name' => $this->getName(),
```

Comment:
Will it be possible to show the project code quality score through this API? You may need to create a custom normalizer as described in https://github.com/algolia/search-bundle#using-a-custom-normalizer